docs/docs/50-user-guide/20-how-to-guides/30-working-with-warehouses.md

```diff
@@ -683,6 +683,16 @@ appropriate credentials must be made available in some form. The many available
 authentication options are covered in detail on the
 [Managing Credentials](../50-security/30-managing-credentials.md) page.
 
+## Automatic Freight Creation Criteria
+
+For more granular control over the creation of your `Freight`, you can use 
+`Freight Creation Criteria` which allows you to define a condition (in the form
+of an expression) that must evaluate to `true` in order to proceed with minting
+new `Freight`. This can be useful if you need to compare information from 
+different artifacts among your `Warehouse` subscriptions. Check out the 
+[Expression Language Reference](../60-reference-docs/40-expressions.md) for more 
+details.
+
```

Comment:
As mentioned, I think we ought not go into great detail about the expressions here. The link you included covers it, but I think an example here would still be a good idea. I think the examples you used in the expression reference doc are good and one could be reused here. (I like the image one best, because it feels like the one that will be most readily understood.) You could introduce the example with some text explaining the hypothetical use case of back and and front end versions always needing to match.